site/en/tutorials/customization/basics.ipynb

```diff
@@ -239,7 +239,7 @@
       "source": [
         "### Device names\n",
         "\n",
-        "The `Tensor.device` property provides a fully qualified string name of the device hosting the contents of the tensor. This name encodes many details, such as an identifier of the network address of the host on which this program is executing and the device within that host. This is required for distributed execution of a TensorFlow program. The string ends with `GPU:<N>` if the tensor is placed on the `N`-th GPU on the host."
+        "The [`Tensor.device`](https://www.tensorflow.org/api_docs/python/tf/Tensor#attributes) property provides a fully qualified string name of the device hosting the contents of the tensor. This name encodes many details, such as an identifier of the network address of the host on which this program is executing and the device within that host. This is required for distributed execution of a TensorFlow program. The string ends with `GPU:<N>` if the tensor is placed on the `N`-th GPU on the host."
```

Comment:
Good idea. `Tensor.device` on its own creates an auto-link for the site pointing to "https://www.tensorflow.org/api_docs/python/tf/Tensor#device", which doesn't work. 

Shall we manually try this "https://www.tensorflow.org/api_docs/python/tf/Tensor#attributes" as per @RenuPatelGoogle 's suggestion @MarkDaoust @markmcd ?